src/dji_osdk_ros/samples/msdk_osdk_node.cpp

```diff
@@ -106,12 +115,44 @@ class MobileCommandHandler
             return;
         }
 
+        if (!testOSDKActivation())
```

Comment:
later: move this to the correct place; drop should not depend on OSDK activation - only sending missions

Reply:
moved to correct spot to enable drop